client/src/components/Calendar.js

```diff
@@ -1,25 +1,49 @@
-import React, { Component } from 'react';
-import Calendar from 'react-calendar';
-import 'react-calendar/dist/Calendar.css';
+import React, { createRef, useState } from 'react';
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import "@fullcalendar/core/main.css";
+import "@fullcalendar/daygrid/main.css";
+import "@fullcalendar/timegrid/main.css";
+import "../css/Calendar.css";
 
-class AppCalendar extends Component { 
-  state = {
-    date: new Date(),
-  }
 
-  onChange = date => this.setState({ date })
+const AppCalendar = () => { 
+
+  // state emulation, store all events user created
+  const [calendarEvents, setCalendarEvents] = useState([])
 
+  // create new event function
+  const handleCreateEventClick = (arg) => {
+    let arrayObject = {
+      title: 'new event',
+      start: arg.date,
+      allDay: arg.allDay
+    }
+    setCalendarEvents([...calendarEvents, arrayObject])
+  }
```

Comment:
I think using `useState` is fine for now as this Calendar is more for aesthetic purposes for now, but in the future I'm just noting that we may refactor this to implement Redux, as we might be storing some information here that might be passed through multiple components

Reply:
I'm more then happy to refactor this component with useSelect when we will have necessary data points in the redux store and useDispatch action to update the store.